README.md

```diff
@@ -12,4 +12,4 @@ and use it 3 times inside the `App` instead of static markup.
 ## Instructions
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Open one more terminal and run tests with `npm test` to ensure your solution is correct.
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_person/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://Krivets.github.io/react_person/) and add it to the PR description.
```

Comment:
You should add this link to the PR description 
![image](https://github.com/mate-academy/react_person/assets/99403198/fdd7818a-27f7-4209-8a15-73abe2eee3dd)